src/style.css

```
@@ -1 +1,75 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin: 0 auto;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.logo {
+  display: inline-block;
```

You have a flex wrapper around, you don't need inline-block here, you can just make it block
```suggestion
  display: block;
```